caravel/templates/moderation/view.html

```diff
@@ -0,0 +1,60 @@
+{% extends "layout.html" %}
```

Comment:
Thoughts on swapping out the individual approve button for a bunch of checkboxes and a "Submit for approval" button below?

Makes it harder to batch approve as it stands.